apps/webapp/src/components/shared/toast-content.tsx

```diff
@@ -86,7 +86,10 @@ export const broadcastingTxToast = (
   };
 };
 
-export const successTxToast = (txHash: string, detectionHeight?: bigint): ToastFnProps => ({
+export const successTxToast = (
+  txHash: string,
+  detectionHeight?: bigint | undefined,
```

Comment:
`exactOptionalPropertyTypes` is disabled now! so you don't need this 🥳

Reply:
gonna defer removal of optional |undefineds to another issue